CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.7)
+project(darkstar)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+include(cmake/compiler.cmake)
+
+if(UNIX)
+    set(lib_path /usr/lib /usr/local/lib /opt/local/lib)
```

Comment:
This isn't really good. Should use pkg-config, hardcoding paths like this is asking for trouble. The cmake Find* modules aren't very good either.

Reply:
https://cmake.org/cmake/help/v3.0/module/FindPkgConfig.html

Reply:
I had not seen this as a way for checking for libs/include directories.  I like it.